unime/.prettierignore

```diff
@@ -5,6 +5,7 @@ node_modules
 /package
 .env
 .env.*
+.tsconfig.json
```

Comment:
Did you have problems `tsconfig` being formatted differently on your machine?

Reply:
Yes it adds trailing slashes on that file, which will fail again on github actions

Reply:
The file `.tsconfig.json` does not exist. You might mean `tsconfig.json`, but I suggest you undo that ignore and reintroduce it if you encounter issues again.